commisery/rules.py

```diff
@@ -473,6 +479,31 @@ def C020_git_trailer_contains_whitespace(
             )
 
 
+def C021_description_should_start_with_alphabetic_character(
+    message: CommitMessage, config: Configuration
+):  # pylint: disable=C0103
+    """The commit message's description must start with an alphabetic character"""
+    # No need to verify merge commits
+    if message.is_merge():
+        return
+
+    try:
+        # No need to run this rule in case the description is not present
+        C009_missing_description(message, config)
+    except logging.Error:
+        return
```

Comment:
Theoretically one could have had this rule (C009) disabled in the configuration

Reply:
But this rule *requires* that one. So you could just add to the documentation for either that or this option that it will force that one to be enabled.

Reply:
In the end - this dependency between tests was not needed at all.